packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/android/githubinstructions.md

```diff
@@ -1,2 +1,6 @@
 After you satisfy the software requirements, clone the [okta-idx-android](https://github.com/okta/okta-idx-android)
-repository to your local directory.
+repository to your local directory. You run the sample app later after you [set the configuration values](/docs/guides/oie-embedded-common-download-setup-app/-/main/#set-the-configuration-values).
+
+The Okta Identity Engine Android SDK project consists of the following notable directory:
```

Comment:
I think these two sentences could be combined into one, for easier reading, and less bitty phrasing/structure. For example "The Identity Engine Android SDK project contains the `okta-idx-android/app` directory, inside which you will find the files for the embedded authentication with SDK app sample."